### ISSUE TITLE
Thonny doesn't work properly 

### ISSUE THREAD

**Issue by MonkaBang** [pop-os/cosmic-epoch#623]
Specifically, the top bar's menus don't deploy when clicked. Not sure if cosmic issue, used to work fine on 22.04

**Comment by rayhankimi** [comment]
yeah i got the same issues after installing cosmic session in 22.04
